docs/Developers/Writing_Good_Commits.rst

```diff
@@ -115,7 +115,7 @@ Example:
 
 ::
 
-    Fixes https://github.com/coala/coala/issues/269
+    Closes https://github.com/coala/coala/issues/269
```

Comment:
Yeah, so basically this needs to be restructured with an actual issue, that should be addressed by `Closes`, with the required commit shortlog, body and an issue reference or maybe we could take into consideration the `Quick reference` example and split it into shortlog, message and issue reference. @prateeksingh0001